.editorconfig

```diff
@@ -0,0 +1,20 @@
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+
+# WordPress Coding Standards
+# https://make.wordpress.org/core/handbook/coding-standards/
+
+# Customized for 2 space indentation and Markdown files
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 4
+
+[{*.md,*.markdown}]
+trim_trailing_whitespace = false
```

Comment:
this is your own configuration, should not be commited. Or you can place in another location instead of this repo.